Sming/project.mk

```diff
@@ -430,6 +430,23 @@ $(foreach c,$(BUILDABLE_COMPONENTS),$(eval $(call GenerateComponentTargets,$c)))
 # The arch-specific targets, including link and firmware image creation
 include $(ARCH_BASE)/app.mk
 
+.PHONY: vscode
```

Comment:
In the future we could have support for more Integrated Development Environments (IDE)s. How about:
1. Adding to `make help` a new topic called "IDE tools"
2. Rename the `vscode*` targets to `ide-vscode`. This way the autocompletion in bash can hint the user to find easily all supported `ide` operations by typing `make ide[PRESS-TAB]`